### PR TITLE
top 5 custom node examples

### PR DESCRIPTION
**Pull request by kning**
Corresponding code examples for a blog post i'm working on called "the 5 best comfyui custom nodes". I think the code samples go here so we can leverage synthetic monitoring tests. will share the blog post copy separately with folks

### Type of Change

- [x] New example
- [ ] Example updates (Bug fixes, new features, etc.)
- [ ] Other (changes to the codebase, but not to examples)

## Checklist

- [x] Example is testable in synthetic monitoring system, or `lambda-test: false` is added to example frontmatter (`---`)
  - [x] Example is tested by executing with `modal run` or an alternative `cmd` is provided in the example frontmatter (e.g. `cmd: ["modal", "deploy"]`)
  - [x] Example is tested by running with no arguments or the `args` are provided in the example frontmatter (e.g. `args: ["--prompt", "Formula for room temperature superconductor:"]`
- [x] Example is documented with comments throughout, in a [_Literate Programming_](https://en.wikipedia.org/wiki/Literate_programming) style.
- [x] Example does _not_ require third-party dependencies to be installed locally
- [ ] Example pins its dependencies
  - [x] Example pins container images to a stable tag, not a dynamic tag like `latest`
  - [x] Example specifies a `python_version` for the base image, if it is used
  - [x] Example pins all dependencies to at least minor version, `~=x.y.z` or `==x.y`
  - [x] Example dependencies with `version < 1` are pinned to patch version, `==0.y.z`

## Outside contributors

You're great! Thanks for your contribution.

### CODE REVIEW

**Comment by charlesfrye** [comment]
🚀 The docs preview is ready! Check it out here:     https://modal-labs-examples--frontend-preview-8e15ece.modal.run

File: 06_gpu_and_ml/comfyui/impact/impact_example.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    .run_commands(  # use comfy-cli to install the ComfyUI repo and its dependencies
+        "comfy --skip-prompt install --nvidia"
+    )
+    .run_commands(  # download the Impact pack
+        "comfy node install ComfyUI-Impact-Pack"
+    )
+    .pip_install("ultralytics==8.3.26")  # object detection models
+    .apt_install(  # opengl dependencies
+        "libgl1-mesa-glx", "libglib2.0-0"
+    )
```
Comment (kramstrom) [inline 06_gpu_and_ml/comfyui/impact/impact_example.py:24]: Nice, I think it's helpful to have this to show that for some of these custom nodes you need to install other dependencies as well

**Review by kramstrom — APPROVED** [review]
looking forward to the blog post!

**Comment by charlesfrye** [comment]
🚀 The docs preview is ready! Check it out here:     https://modal-labs-examples--frontend-preview-1a32261.modal.run

**Comment by charlesfrye** [comment]
🚀 The docs preview is ready! Check it out here:     https://modal-labs-examples--frontend-preview-cd83b40.modal.run

**Pull request merged**